homeassistant/components/demo/strings.json

```diff
@@ -63,6 +63,29 @@
     }
   },
   "entity": {
+    "climate": {
+      "demo_climate": {
+        "attribute": {
+          "fan_mode": {
```

Comment:
There's also a `fan_modes` attribute, do we need to translate that too or is it enough that frontend knows that the two are connected?

Reply:
I don't think that is needed. This specific case is handled by the UI in a special way already. (Climate)

I look at it a similar way as the `select` entity, for which we have a `state` in the translation, but the list is provided in the `options` attribute.